include/mbedtls/mbedtls_config.h

```diff
@@ -641,11 +639,10 @@
  *        mbedtls_ssl_handshake(), can now return
  *        MBEDTLS_ERR_SSL_CRYPTO_IN_PROGRESS.
  *
- * \note  When this option and MBEDTLS_USE_PSA_CRYPTO are both enabled,
- *        restartable operations in PK, X.509 and TLS (see above) are not
- *        using PSA. On the other hand, ECDH computations in TLS are using
- *        PSA, and are not restartable. These are temporary limitations that
- *        should be lifted in the future.
+ * \note  When this option is enabled, restartable operations in PK, X.509
+ *        and TLS (see above) are not using PSA. On the other hand, ECDH
+ *        computations in TLS are using PSA, and are not restartable. These
+ *        are temporary limitations that should be lifted in the future.
```

Comment:
I am sorry I am not sure I can see which of these limitations has gone. Could you please elaborate?

Reply:
Raised https://github.com/Mbed-TLS/mbedtls/issues/9817 for tracking.

Reply:
Sorry, I must have forgotten to actually write a sentence here, or I made a mistake when consolidating/separating comments. Grammatically, the current paragraph contains two limitations (one per sentence). The first limitation “… not using PSA” is not really meaningful anymore, since users must assume that everything is “using PSA” — users must call `psa_crypto_init` and must enable `PSA_WANT_xxx`. So I thought we should remove that sentence: it's now an implementation detail, no longer a statement about the interface. The sentence isn't misleading though, and it's informative in a non-semantic way, so it can stay.